src/vs/platform/sharedProcess/electron-browser/sharedProcessWorkerService.ts

```diff
@@ -322,7 +322,7 @@ class SharedProcessWebWorker extends Disposable {
 			// working directory. Setting the env BREAKPAD_DUMP_LOCATION will force electron to create the file in that location,
 			// For https://github.com/microsoft/vscode/issues/105743
 			const extHostCrashDirectory = this.environmentService.args['crash-reporter-directory'] || this.environmentService.userDataPath;
-			sharedProcessWorkerEnvironment.env.BREAKPAD_DUMP_LOCATION = join(extHostCrashDirectory, this.type);
+			sharedProcessWorkerEnvironment.env.BREAKPAD_DUMP_LOCATION = join(extHostCrashDirectory, `Parcel Watcher Crash Reports`);
```

Comment:
@deepak1556 whats the reason for this change? the shared process worker service actually shouldn't hardcode parcel knowledge, we may use it for other processes too.

Reply:
@bpasero you might be looking at an outdated diff, breakpad code is no longer needed in this branch, I have actually removed these changes.

Reply:
Oh yeah.